# W01 — Research Question
## Section 1 — My Lane and Why

**Lane chosen: Lane 2 — Refresh / Content Opportunity Scoring**

Content teams manage hundreds of pages but only have time to review 20 to 50
per week. Without a system, deciding which pages to look at first relies on
simple hand-written rules. My project builds a ranked review


## Section 2 — The Question: Decision, Action, Cost of a Wrong Call

**The Decision:** Reviewers must select a prioritized subset of pages from
hundreds of potential candidates to maximize the impact of their limited
weekly review hours.

**The Action:** For each high-priority page, reviewers take a specific action
such as updating stale content, expanding thin pages, or setting up monitoring.

**The Cost of a False Alarm:** Flagging a healthy page mistakenly wastes
valuable analyst time. This can drain a full week of productivity across
multiple errors while actual issues go unnoticed.

**The Cost of a Missed Page:** Failing to identify a declining page allows
traffic and impressions to drop silently. This leads to a much deeper and
harder to recover ranking loss over time.

In [2]:
import pandas as pd
df = pd.read_csv("content_refresh_anonymized.csv")
print(df.shape)

(30000, 44)


In [7]:
import pandas as pd
df = pd.read_csv("content_refresh_anonymized.csv")

total = len(df)
declining = (df['trend_direction'] == 'down').sum()
print(f"{declining} out of {total} pages are declining ({round(declining/total*100, 1)}%)")
print(f"Average page age: {round(df['content_age_days'].mean(), 1)} days")
print(f"Average age of declining pages: {round(df[df['trend_direction'] == 'down']['content_age_days'].mean(), 1)} days")

16262 out of 30000 pages are declining (54.2%)
Average page age: 256.2 days
Average age of declining pages: 236.2 days


"Because the average page is 8+ months old and over half are declining, there is a large backlog of content that needs attention - and without a ranked system, reviewers have no way to know which ones to look at first.

**The dataset has 30,000 pages with an average age of 256 days. Over half  54.2%  are currently declining. Interestingly, declining pages have almost the same average age as the rest, which means age alone does not predict which pages need attention. This is why a simple hand-written rule is not enough and a model that learns from multiple signals at once is needed to rank pages by how urgently they need review.**

## Section 4 — Careful Words: What I Can and Can't Claim

This project produces a ranked list of pages that may need attention; it does
not guarantee that reviewed pages will actually be updated. The findings are
based on one company's anonymized data and may not apply to other organizations
or industries. The model describes current trend direction, not future outcomes,
where a stronger version would use past data to predict future performance.

## Section 5 — Self-Check

1. **Specific decision?** Yes - a reviewer decides which pages to prioritize
   for review each week from a pool of thousands.

2. **Action named?** Yes - the reviewer opens high-priority pages and takes
   a specific action: refresh, expand, prune, or flag for monitoring.

3. **Cost of a wrong call explained?** Yes - missing a declining page allows
   it to keep losing traffic, making recovery harder over time.

4. **More than just "train a model"?** Yes - the output is a ranked review
   queue that helps a real person make better use of limited review time,
   ultimately protecting the client's search visibility.